tools/bumper/cnao_repo_commands.go

```diff
@@ -212,6 +215,138 @@ func (cnaoRepoOps *gitCnaoRepo) collectModifiedToTreeList(allowedList []string)
 	return entries, nil
 }
 
+// getBumpBranch creates a new bump branch.
+// Since we don't want to use already open branch
+// that may hold outdated/needs rebasing changes
+// we concat a random string to the end of the branch name
+func (cnaoRepoOps *gitCnaoRepo) getBumpBranch(prTitle string) (*github.Reference, string, error) {
+	branchBaseName := strings.Replace(strings.ToLower(prTitle), " ", "_", -1)
+	branchOpenMaxTries := 10
+	for branchIdx := 0; branchIdx < branchOpenMaxTries; branchIdx++ {
```

Comment:
`s/branchIdx/branchOpenAttempts/g`

Do we really need to retry ?

Reply:
since it's pseudo random it'll get the same name, but it should be fixed with seed. 
Done